.github/workflows/integrate.yaml

```diff
@@ -60,7 +60,10 @@ jobs:
         microk8s-addons: "dns storage rbac metallb:10.64.140.43-10.64.140.49"
 
     - name: Run integration tests
-      run: tox -vve integration -- --model testing
+      run: tox -ve integration -- --model testing
+
+    - name: Run workload integration tests
+      run: tox -ve workload-integration -- --model testing
```

Comment:
will that play nice with reusable CI?

do not think we have workload-integration anywhere else

Reply:
Will put it back into single integration tests.
But having separate workload tests is a good idea.